tests/containers/podman_integration.pm

```diff
@@ -64,7 +64,7 @@ sub run {
     }
 
     # Install tests dependencies
-    my @pkgs = qw(aardvark-dns bats catatonit jq make netavark netcat-openbsd openssl python3-PyYAML socat sudo systemd-container);
+    my @pkgs = qw(aardvark-dns bats catatonit jq make netavark netcat-openbsd openssl passt python3-PyYAML socat sudo systemd-container);
```

Comment:
Isn't that needed only on podman version 6+? Wouldn't that mean that we need to add a clause, installing this only on TW atm?

Reply:
> Isn't that needed only on podman version 6+? Wouldn't that mean that we need to add a clause, installing this only on TW atm?

It's required on podman 5 but the tests on 4.9.3 already use it:

https://openqa.suse.de/tests/13913863#step/podman/60

I added a condition for TW, Leap 15.6+ & SLE 15-SP6+.  Perhaps missing SL-Micro & MicroOS.